### DISCUSSION TITLE
Scheduled posts not appearing with "half" caching

### DISCUSSION THREAD

**Discussion by hotmeteor** [statamic/cms#7679] [Troubleshooting]
Hi, I'm trying to use the "Date" field with time enabled to schedule future content. I've also got "half" static caching enabled using the `application` driver.

My content never appears on its own at the correct time, I __always__ have to go and re-save/publish for it to show up on my home page. Here's what I've done with no success:

- Used the `{{ nocache }}` tag around the content
- Added a scheduled job running to clear the cache
- Ensured I'm invalidating the correct pages when the content is saved

The only thing that works is resaving the content (I guess the last bullet is working).

Anyway, am I missing something or does this just not work?

**Comment by joshuablum** [comment]
Hey @hotmeteor,

Looks like this is related to https://github.com/statamic/cms/pull/5089.

You can use [composer-patches](https://rias.be/blog/using-composer-patches) and give the PR a try to see whether this resolves the issue 👍

**Reply by hotmeteor** [reply]
Thanks @joshuablum! I'll give it a shot. However, after reading the comments I should mention I'm using Eloquent so not sure if it'll do the trick or not. I'll let you know.

**Reply by hotmeteor** [reply]
Yeah... no dice. I see the records being updated in the DB but the caching isn't refreshing the content.

- Homepage still shows future-timestamped posts after re-publishing
- Page is still accessible after re-publishing

One thing that's interesting to note is that the caching **is** refreshed the FIRST time you re-publish, but then if you change the timestamp again and re-publish it doesn't. So maybe the logic isn't catching the differences there somehow?
